data/SugarBean.php

```diff
@@ -3266,7 +3266,7 @@ public function send_assignment_notifications($notify_user, $admin)
                     $GLOBALS['log']->fatal("Notifications: error sending e-mail (method: {$notify_mail->Mailer}), " .
                         "(error: {$notify_mail->ErrorInfo})");
                 } else {
-                    $this->sentAssignmentNotifications = true;
+                    $this->sentAssignmentNotifications[] = $notify_user->id;
```

Comment:
Hi  @ojs87,

Thank you for the pull request.

`MeetingTest.php` and  `CasetTest.php` are expecting this to a boolean. Could you also update the tests please?

Another thing, I think it would be good to declare `sentAssignmentNotifications` as an `array` at the top of the `SugarBean` class. This avoids a notice, that we are probably getting at the moment.

Reply:
Hi Clemente, 
Thanks for the feedback, I'll make the suggested changes.